packages/api/src/controllers/webhook.js

```diff
@@ -45,6 +46,7 @@ const fieldsMap = {
   createdAt: `webhook.data->'createdAt'`,
   userId: `webhook.data->>'userId'`,
   "user.email": `users.data->>'email'`,
+  sharedSecret: `webhook.data->>'sharedSecret'`,
```

Comment:
Should the shared secret even be query-able, if it is write-only? Or would it get filtered for non-admin users?